src/playground/services/invoices/update/logic.clj

```diff
@@ -0,0 +1,18 @@
+(ns playground.services.invoices.update.logic
+  (:require
+   [honeysql.core :as h]
+   [honeysql.helpers :as hh]))
+
+(defn to-update [id name uuid]
+  (-> (hh/update :users)
+      (hh/sset  {:email (str name "@" uuid ".com")})
+      (hh/where [:= :id (Integer/parseInt id)])))
```

Comment:
Do the parsing in the handler.

In an MVC architecture, (C being the handler, and M being the logic), validating/coercing user input is responsbility of the C.

If possible, do it via the spec interceptor rather than `Integer/parseInt`